src/endpoints/mod.rs

```diff
@@ -71,9 +72,12 @@ fn server_info() -> JsonValue {
         testing = false;
     }
 
+    let hostname = gethostname().into_string().unwrap_or("unknown".to_string());
+    const VERSION: &'static str = env!("CARGO_PKG_VERSION");
```

Comment:
Wait, does this really work when not running with "cargo run"?

Reply:
The following worked:

```
$ cargo build
...
$ ./target/debug/aw-server-rust
...
$ curl localhost:5667/api/0/info
{"hostname":"erb-laptop2-arch","testing":true,"version":"aw-server-rust v0.1.0"}
```

Reply:
Added a fallback, see: https://stackoverflow.com/questions/27840394/how-can-a-rust-program-access-metadata-from-its-cargo-package

Reply:
Apparently `env!` fetches the variable at compile-time: https://doc.rust-lang.org/std/macro.env.html